wormhole-foundation/relayer-engine
# Detect and fetch missed VAAs

Testing:

- spawn listener 
- send 1 message
- kill listener after message processed
- send 10 messages
- spawn listener
- send 1 message
- observe that all 11 messages get processed

fixes #21 

next up is #25 which allows checking if we missed a vaa without requiring a new vaa to come in
